Library/Formula/sbcl.rb

```diff
@@ -96,15 +94,14 @@ def install
     end
 
     ENV["INSTALL_ROOT"] = prefix
-    system "sh install.sh"
+    sh "install.sh"
```

Comment:
Using `sh` (from FileUtils) has different semantics that using `system` (which is specialized for formulae). So this should be `system "sh", "install.sh"`.